src/lib/libhtml5.js

```diff
@@ -1347,8 +1347,8 @@ var LibraryHTML5 = {
     }
 
     if (inCenteredWithoutScalingFullscreenMode) {
-      var t = (innerHeight - jstoi_q(canvas.style.height)) / 2;
-      var b = (innerWidth - jstoi_q(canvas.style.width)) / 2;
+      var t = (innerHeight - Number(canvas.style.height)) / 2;
+      var b = (innerWidth - Number(canvas.style.width)) / 2;
```

Comment:
where in the docs does it say this must be a string with only an integer? i couldn't find that on MDN.